### ISSUE TITLE
Seems to bootloop when client starts.

### ISSUE THREAD

**Issue by xqozbd** [Skidamek/AutoModpack#289]
### Bug description

I am experimenting with this mod and I tried joining my local hosted server. It got the mod files and closed the game, I re ran the game and it shows this.
![image](https://github.com/user-attachments/assets/b8e5e02b-39b7-4b2a-907e-c261be72acc6)

I looked in the logs, nothing. I havent changed any configs at all yet.

I did see the other issue about the mods in allowEditinFiles config section, there was no mods thing in that.

### Steps to reproduce

1. Get Server with mods, get client with Automodpack
2. Join server, get updates and close game
3. Start client and it fails

### Expected behavior

To load into the game, and allow me to join the server.

### Actual behavior

After clicking play, it says Automodpack updated.

### Minecraft & Mod Loader versions

1.20.1 Fabric 0.16.5

### Minecraft launcher

Minecraft launcher, and tried the beta curseforge launch.

### Operating system

Windows 11

### AutoModpack version

automodpack-mc1.20.1-fabric-4.0.0-beta18

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [x] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by Skidamek** [comment]
Could you please share you client log and automodpack server config?

**Comment by DuckyXD165** [comment]
The server config from the client side or the local hosted server?

**Comment by DuckyXD165** [comment]
[latest.log](https://github.com/user-attachments/files/17270969/latest.log)
Server config from local hosted server:
[automodpack-server.json](https://github.com/user-attachments/files/17270970/automodpack-server.json)
Server config from the client folder:
[automodpack-server.json](https://github.com/user-attachments/files/17270971/automodpack-server.json)

**Comment by Skidamek** [comment]
Are you manually coping over all mods to mods directory every time it shows that popup?

If not then that's strange because what there should happen is that automodpack should remove all duplicated modpack mods from mods directory and relaunch the game with mods from special automodpack mods directory.

Looks like it fails for some strange reason so you should try do it manually just purge your mods folder on client and put there only automodpack. (You can then add additional mods which are not in server modpack no worries)

**Comment by DuckyXD165** [comment]
Ill try it, ill let you know how it goes

**Comment by DuckyXD165** [comment]
Yea it seemed to work now. Thank you!

**Comment by Skidamek** [comment]
Great i will leave it open as a solution for someone and also that just should not happen it's a bug

**Comment by Xiarno** [comment]
> Great i will leave it open as a solution for someone and also that just should not happen it's a bug

Yeah I had that happened too on ATM10 and this fixed it. I did have to manually place back all clientside mod tho.

**Comment by Skidamek** [comment]
It might have been fixed in beta 19, need testing

**Comment by Talmanaze** [comment]
Had this same error but the problem was caused by the mods optimizer mod. (https://www.curseforge.com/minecraft/mc-mods/mods-optimizer)

After removing that, everything booted correctly.

**Comment by Skidamek** [comment]
Well that would make sense, that mod also does some shenanigans with mods. You likely don't want more mods which would mess with your mods structure while using automodpack as it tries to load fullly specified modpack from your server. If you need to make some modpack optimization do it remotely from server side. I will add this mod to automodpack metadata as conflicting so that no one will try to use both of these mods, thanks for reporting.

**Comment by Skidamek** [comment]
should be completely fixed now
